src/core_plugins/markdown_vis/public/markdown_vis_controller.js

```diff
@@ -11,7 +11,11 @@ marked.setOptions({
 const module = uiModules.get('kibana/markdown_vis', ['kibana', 'ngSanitize']);
 module.controller('KbnMarkdownVisController', function ($scope) {
   $scope.$watch('vis.params.markdown', function (html) {
-    if (!html) return;
+    if (!html) {
```

Comment:
Something like this would be cleaner:

```
if (html) { $scope.html = marked(html); }
$scope.$emit('renderComplete');
```

Plus, then there's no potential of forgetting to emit if other conditions are added.